src/kit/LineChart.tsx

```diff
@@ -22,6 +22,15 @@ import css from './LineChart.module.scss';
 export const TRAINING_SERIES_COLOR = '#009BDE';
 export const VALIDATION_SERIES_COLOR = '#F77B21';
 
+const getScientificNotationTickValues: uPlot.Axis['values'] = (_self, rawValue) => {
+  return rawValue.map((val) => {
+    if (val === 0) return val;
+    return val > 9_999 || val < -9_999 || (0 < val && val < 0.0001) || (-0.0001 < val && val < 0)
```

Comment:
is it possible to make the label format consistent? 
if labels use scientific notation, all labels should be scientific notation vice versa

<img width="90" alt="image" src="https://github.com/determined-ai/determined-ui/assets/109113616/9aa72d2b-0c32-4381-92cd-8fddff41fe8a">

Reply:
replaced the function ✔️